tests/unit_test/data_processor/data_processor_test.py

```diff
@@ -1546,7 +1568,7 @@ def test_get_intents_and_training_examples(self):
         actual = processor.get_intents_and_training_examples("tests")
         assert len(actual) == 17
 
-    def test_delete_intent_no_trainingExamples(self):
+    def test_delete_intent_no_training_examples(self):
```

Comment:
tests for:
1. case insensitive text response (edit and add training example)
2. case insensitive story (edit and add)